src/bci_build/package/node.py

```diff
@@ -58,7 +55,7 @@ def _get_node_kwargs(ver: Literal[16, 18, 20], os_version: OsVersion):
         **_get_node_kwargs(18, OsVersion.SP5), support_level=SupportLevel.L3
     ),
     LanguageStackContainer(
-        **_get_node_kwargs(20, OsVersion.SP6), support_level=SupportLevel.L3
+        **_get_node_kwargs(20, OsVersion.SP5), support_level=SupportLevel.L3
```

Comment:
but we also need to have it for SP6.. I suggest to duplicate this for SP5 and keep the SP6 entry.

Reply:
I was under the mistaken impression that we did not produce language stack containers for sp6 yet.